.github/actions/test_stateless_cluster_linux/action.yml

```diff
@@ -15,6 +15,14 @@ runs:
         sha: ${{ github.sha }}
         target: ${{ inputs.target }}
 
+    - uses: actions/checkout@v3
+    - name: Start UDF Server
```

Comment:
This step should goes after `Test setup`

Reply:
All action have been modified.